lib/sles4sap.pm

```diff
@@ -347,26 +342,10 @@ sub copy_media {
     assert_script_run "mkdir $target";
     assert_script_run "mount -t $proto -o ro $path $mnt_path";
     $media_path = $mnt_path if script_run "[[ -d $media_path ]]";    # Check if specific ARCH subdir exists
-    assert_script_run "cp -ax $media_path/. $target/", $nettout;
+    script_retry ("rsync --archive --checksum $rsync_server:$media_path/. $target/", timeout =>$nettout, delay => 60, retry => 3);
```

Comment:
> @ricardobranco777 @alvarocarvajald WDF tests use HANA_REL variable which specifies directory in $media_path for HANA release being used. For example: /data/HANA2/Last_SPS06 or /data/HANA2/Last_SPS05 One option would be just appending variable after $media_path/HANA_REL. Second option could be to have if statement which will do the 'cp' in case HANA_REL is defined, otherwise rsync.

But **HANA_REL** is already appended to **HANA**, for example: https://gitlab.suse.de/qa-css/openqa_ha_sap/-/blob/master/openQA/yaml_job_groups/openqa.wdf.sap.corp/sles4sap-15sp3.yml#L30

And **HANA** in turn points to a NFS share.

So, if it's possible to have an rsync server there, then the **HANA** setting would need to be changed. Otherwise, mounting over NFS and then copying with `rsync` as @ricardobranco777 is proposing here, should be safe.

IMO, before switching completely to rsync, a rsync server needs to be configured in WDF and rsync connectivity to it from all openQA workers and ppc64le LPARs need to be confirmed to be working. If rsync is somehow blocked by an ACL, then I would say it is better to keep NFS in the picture, as we know that protocol is allowed. An hybrid approach, i.e., rsync for osd, and NFS for everywhere else could also be possible, but I feel like that makes things unnecessarily complicated.